exams/api.py

```diff
@@ -32,3 +57,115 @@ def sso_digest(client_candidate_id, timestamp, session_timeout):
         str(client_candidate_id),
     ]).encode('iso-8859-1')
     return hashlib.sha256(data).hexdigest()
+
+
+def authorize_for_exam_run(mmtrack, course_run, exam_run):
```

Comment:
Does `exam_run.course` need to be the same as `course_run.course`? If so, can we just obtain `exam_run` from `course_run`?